libs/unity/library/Runtime/Scripts/NativeRender/NativeVideo.cs

```diff
@@ -84,23 +95,50 @@ public void EnableRemoteVideo(VideoKind format, TextureDesc[] textures)
 
             NativeVideoInterop.EnableRemoteVideo(_nativeVideoHandle, format, interopTextures, interopTextures.Length);
         }
+        
+        public void UpdateTextures(TextureDesc[] textures)
+        {
+            var interopTextures = textures.Select(item => new NativeVideoInterop.TextureDesc
```

Comment:
Related to other comment, should we validate the size of `textures[]` here? If it's smaller than 3 we will have an out-of-bound access in native.

Reply:
Added validation method.